docs/en/operations/storing-data.md

```diff
@@ -172,37 +172,41 @@ Example of configuration for versions earlier than 22.8:
     </storage_configuration>
 ```
 
-Cache **configuration settings**:
+File Cache **configuration settings**:
+
+These settings should be defined in the disk configuration section.
 
 - `path` - path to the directory with cache. Default: None, this setting is obligatory.
 
 - `max_size` - maximum size of the cache in bytes or in readable format, e.g. `ki, Mi, Gi, etc`, example `10Gi` (such format works starting from `22.10` version). When the limit is reached, cache files are evicted according to the cache eviction policy. Default: None, this setting is obligatory.
 
-- `cache_on_write_operations` - allow to turn on `write-through` cache (caching data on any write operations: `INSERT` queries, background merges). Default: `false`. The `write-through` cache can be disabled per query using setting `enable_filesystem_cache_on_write_operations` (data is cached only if both cache config settings and corresponding query setting are enabled).
```

Comment:
why remove this sentence?

Reply:
I deleted it because I was explaining below:
```
Some of these settings will disable cache per query features that are enabled by default. For example, setting `cache_on_write_operations` to 1 means that general file and per query cache are enabled but also setting `enable_filesystem_cache_on_write_operations` to 1 means that file cache is enabled but disabled per query cache. The same approach must be used for `enable_filesystem_cache`, if set to 1, cache per query is disabled.
```
but I'll put it back and align both paragraphs.